tests/test_smoke.py

```diff
@@ -1506,20 +1509,26 @@ def _get_cancel_task_with_cloud(name, cloud, timeout=15 * 60):
 
 # ---------- Testing `sky cancel` ----------
 @pytest.mark.aws
+@pytest.mark.skip(
```

Comment:
Hmm, it seems a bit dangerous to skip these core tests? Maybe we should endure the flakiness for now?

Reply:
Should we instead fix #2207 ? It seems to be problematic tests.

Reply:
should we mark it as P0? To avoid skipping these core tests for too long.